doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -343,6 +344,8 @@ Matter samples
   * Fixed an issue that prevented nRF Toolbox for iOS in version 5.0.9 from controlling the sample using :ref:`nus_service_readme`.
   * Changed the design of the :ref:`matter_lock_sample_wifi_thread_switching` feature so that support for both Matter over Thread and Matter over Wi-Fi is included in a single firmware image.
 
+* Disabled :ref:`ug_matter_configuring_read_client` in most Matter samples using the new :kconfig:option:`CONFIG_CHIP_ENABLE_READ_CLIENT` Kconfig option.
```

Comment:
Just realized this should come before the Door Lock sample information and after the unification of buttons and LEDs that is not present in this PR. I think this will require a rebase to avoid merge conflicts.

Reply:
Moved, and rebased :)